src/scss/cs/competition.scss

```diff
@@ -131,18 +130,52 @@
   }
 
   &-background-1 &-arrow {
-    border-left-color: #F9F9F9;
+    border-left-color: #f9f9f9;
   }
 
   &-background-2 &-arrow {
-    border-left-color: #FFBBDC;
+    border-left-color: #ffbbdc;
   }
 
   &-background-3 &-arrow {
-    border-left-color: #FFDF80;
+    border-left-color: #ffdf80;
   }
 
   &-background-4 &-arrow {
-    border-left-color: #FFC000;
+    border-left-color: #ffc000;
+  }
+}
+
+.entry-form {
+  &-section {
+    width: 100%;
+    background-color: #ffdf80;
+  }
+
+  &-background-img {
+    background-image: url("../../../public/assets/CompetitionTrianglePattern.svg");
+    background-repeat: no-repeat;
+    background-position: left;
+    background-size: contain;
+  }
+
+  &-title {
+    color: #120540;
+    font-size: 36px;
+  }
+
+  &-sub-title {
+    color: #130019;
+    font-size: 18px;
+    font-family: "dm-sans-light", DMSans-Light, sans-serif;
+  }
+
+  &-button {
+    background-color: #d60265;
+    color: white !important;
+  }
+
+  &-button-label{
+    padding-top: 34px;
```

Comment:
Same here for using bootstrap instead of css as well :)

Reply:
The reason why it is being used as css as it is not available to get close to padding 34 from bootstrap, its either too big or small. its between p 4 and 5.